app/src/main/java/com/sopt/android_hyebin/util/BaseFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.sopt.anroid_hyebin.util
+
+import android.app.Dialog
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+
+abstract class BaseFragment<T : ViewDataBinding>(@LayoutRes val layoutRes: Int) : Fragment() {
```

Comment:
class 내부에서만 쓰일 변수라면

```suggestion
abstract class BaseFragment<T : ViewDataBinding>(@LayoutRes private val layoutRes: Int) : Fragment() {
```

private 접근자 달아주기~!

Reply:
넵! 감사합니당 :)